data/map.txt

```diff
@@ -27920,6 +28134,14 @@ planet Mainsail
 		threshold 3500
 		fleet "Small Republic" 57
 
+planet Mahoganybox
+	attributes station hai retirement
+	landscape land/space6
+	description `A visitor to this station will immediately notice that the gravity is lower than most planets by about a third. Leaving the docking bay brings one into a expansive room that can only be described as stately. A round reception desk greets you just inside, and is situated such that the occupants can face any direction that suits them and observe the room. A smiling Hai in a suit informs unless you have family at this respite facility, you will not be permitted to wander beyond this room. However, you are welcome to visit with anyone who has come down, or to ask an orderly if you need to contact someone on business.`
```

Comment:
```suggestion
	description `A visitor to this station will immediately notice that the gravity is lower than most planets by about a third. Leaving the docking bay brings one into an expansive room that can only be described as stately. A round reception desk greets you just inside, and is situated such that the occupants can face any direction that suits them and observe the room. A smiling Hai in a suit informs unless you have family at this respite facility, you will not be permitted to wander beyond this room. However, you are welcome to visit with anyone who has come down, or to ask an orderly if you need to contact someone on business.`
```

Reply:
I'll add a couple of my own here as well.
```suggestion
	description `A visitor to this station will immediately notice that the gravity is lower than most planets by about a third. Leaving the docking bay brings one into an expansive room that can only be described as stately. A round reception desk greets you just inside, and is situated such that the occupants can face any direction that suits them and observe the room. A smiling Hai in a suit informs you that unless you have family at this respite facility, you will not be permitted to wander beyond this room. You are, however, welcome to visit with anyone who has come down or ask an orderly if you need to contact someone on business.`
```